Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -778,6 +781,15 @@ YUI.add('ez-platformuiapp', function (Y) {
                     view: "sectionServerSideView",
                     callbacks: ['open', 'checkUser', 'handleSideViews', 'handleMainView']
                 }, {
+                    name: "adminRole",
```

Comment:
Noticed this first now, but this one is now duplicated with the one a bit further below, this is the right one now, the other should be removed.